google-cloud-spanner-cassandra/src/main/java/com/google/cloud/spanner/adapter/Launcher.java

```diff
@@ -83,6 +131,8 @@ public static void main(String[] args) throws Exception {
     final String maxCommitDelayProperty = System.getProperty(MAX_COMMIT_DELAY_PROP_KEY);
     final boolean enableBuiltInMetrics =
         Boolean.parseBoolean(System.getProperty(ENABLE_BUILTIN_METRICS_PROP_KEY, "false"));
+    final int healthCheckPort =
```

Comment:
might be better to make this an Integer and check for null instead of 0 in start()?